scripts/main.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python3
+import typer
+from pathlib import Path
+import subprocess
+from typing import Optional
+from rich.console import Console
+from rich import print as rprint
+from enum import Enum
+
+app = typer.Typer(help="LAN Pipeline CLI tools")
+console = Console()
+
+
+class Backend(str, Enum):
```

Comment:
high level comment but just putting it here:

I think overall let's focus on making the `jax` part work properly everywhere, since this is what is currently actively used via HSSM. If pytorch doesn't work initially, it's not going to do any harm for any of our current core workflows.

The legacy aspect of this is that I had first done stuff in tensorflow/keras, then refactored to do everything in torch (by the time I started on LanFactory it was torch), then added jax.